Source/Libraries/NetCore/TCPLink.cs

```diff
@@ -61,14 +61,7 @@ internal TCPLink(NetCoreSpec _spec)
             DefaultBoopMonitoringCounter = spec.DefaultBoopMonitoringCounter;
             BoopMonitoringCounter = spec.DefaultBoopMonitoringCounter;
 
-            if (spec.AutoReconnect)
```

Comment:
I'm honestly surprised how much redundant code is in here sometimes. It looks like this overlaps with the reconnect code within the RTC. Can you add a comment noting the removal of this and pointing to this issue just for reference?

Reply:
Added a comment here so if someone ctrl+F's "StartNetworking()" they'll see this.